src/components/GitmojiList/index.js

```diff
@@ -32,6 +36,39 @@ const GitmojiList = (props: Props): Element<'div'> => {
       })
     : props.gitmojis
 
+  const addPinned = (code: string): void => {
+    setPinneds([...pinneds, code])
+    notify(
+      `<p>Gitmoji <span class="gitmoji-code">${code}</span> pinned to the top 😜</p>`
+    )
+  }
+
+  const removePinned = (code: string): void => {
+    setPinneds(pinneds.filter((pinned) => pinned !== code))
+    notify(
+      `<p>Gitmoji <span class="gitmoji-code">${code}</span> is unpinned 😜</p>`
+    )
+  }
+
+  const isPinned = (code: string): boolean => {
+    return pinneds.includes(code)
+  }
+
+  const sortPinnedGitmojis = (gitmojis: Array<Gitmojis>): Array<Gitmojis> => {
+    return gitmojis.sort((gitmoji) => {
+      if (isPinned(gitmoji.code)) return -1
+      return 0
+    })
+  }
+
+  const onPinClick = (code: string): void => {
+    if (isPinned(code)) {
```

Comment:
IF you like the `pinDictionary` suggestion:

```suggestion
    if (Boolean(pinDictionary[gitmoji.code])) {
```